src/test/java/org/mockitousage/stubbing/StubbingWithAdditionalAnswersTest.java

```diff
@@ -56,12 +58,114 @@ public void can_return_arguments_of_invocation() throws Exception {
         given(iMethods.objectArgMethod(any())).will(returnsFirstArg());
         given(iMethods.threeArgumentMethod(eq(0), any(), anyString())).will(returnsSecondArg());
         given(iMethods.threeArgumentMethod(eq(1), any(), anyString())).will(returnsLastArg());
-        given(iMethods.mixedVarargsReturningString(eq(1), any())).will(returnsArgAt(2));
 
         assertThat(iMethods.objectArgMethod("first")).isEqualTo("first");
         assertThat(iMethods.threeArgumentMethod(0, "second", "whatever")).isEqualTo("second");
         assertThat(iMethods.threeArgumentMethod(1, "whatever", "last")).isEqualTo("last");
-        assertThat(iMethods.mixedVarargsReturningString(1, "a", "b")).isEqualTo("b");
```

Comment:
Is this the only stub that would now start failing?

Reply:
Unintentional removal - I'll add it back in.

Reply:
Ah great! In that case, if all existing tests pass and these additional cases also pass, I am up for this! I have not reviewed your implementation yet, but as long as we can keep all existing regression tests passing I am happy 😄